tests/dummy/app/components/my-auto-complete.js

```diff
@@ -16,5 +16,5 @@ export default AutoComplete.extend({
           caseInsensitiveOptions.push(value.toLowerCase());
       });
       return caseInsensitiveOptions;
-  }.property("options.@each")
+  }.property("options.[]")
```

Comment:
I see this change from options.@each to options.[] but I don't know what one offers over the other. You have a blog post I could read/source link to ember that explains this as a win/win ?